### ISSUE TITLE
handling 3rd party launchers development

### ISSUE THREAD

**Issue by Vasil3333** [ValveSoftware/Proton#6438]
# Compatibility Report
- Name of the game with compatibility issues: GTA 5, Red Dead Redemption 2 , ubisoft games , ea games...

## System Information
- GPU: GTX 970
- Driver Version:  4.6.0 NVIDIA 525.60.11
- OpenGL Version: 4.6
- Kernel version: 5.15.85-1-MANJARO
- Proton version: Experimental , 7.0-5, 6.3.8, 5.13-6, 5.0-10

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 
Games don't start or fail to initialize properly because of their 3rd party launchers because 3rd party launchers try to connect to some servers and get like a validation information.

All games that i own and that have an external launcher have always been a hit or miss with proton. And what works today might not work tomorrow which makes the whole proton/linux experience a failure, since if i want to play a specific game , i want it to work NOW not maybe half a year later.

Proton needs heavy development in the not so "3d" space.
Basically do what actions the launchers want to do.

- GTA 5 for example launches the game after the launcher sits for 1 min searching for something and then when entering game it doesnt allow me to play because I am in offline mode and it cant validate my game to allow me to play. (1 month ago it didn't even start , the launcher would give me an error with 'failed to start online mode and fallback of offline also failed' and it wouldnt even try to start the game.
- Red Dead Redemption 2 doesn't start at all. The launcher just opens and then say that i need to connect to the internet.
- EA and ubisoft games also are hit or miss and have left me in the dust in the most important moments.

## Reproduction
Try to start the game

**Comment by kisak-valve** [comment]
Hello @Vasil3333, support for the changing requirements of multiple third party launchers can't be generalized into a single topic and needs to be evaluated on a per-game basis. Feel free to add your experience to the individual compatibility reports of games which you're seeing issues along with logs and your system information.
